Docker CMake Error

OS: Ubuntu 24.04
Docker: 27.1.2
QGC: master

When running `./deploy/docker/run-docker.sh` I receive the following Errors:
`ninja: error: loading 'build.ninja': No such file or directory`
`CMake Error: Not a file: /project/build/cmake_install.cmake`
`CMake Error: Error processing file: /project/build/cmake_install.cmake`
Here is the full log:
[docker.log](https://github.com/user-attachments/files/16750297/docker.log)

--- a/deploy/docker/Dockerfile-build-ubuntu
+++ b/deploy/docker/Dockerfile-build-ubuntu
@@ -23,6 +23,8 @@ RUN /tmp/qt/install-qt-debian.sh
 # Reconfigure locale
 RUN locale-gen en_US.UTF-8 && dpkg-reconfigure locales
 
+RUN git config --global --add safe.directory /project/source
+
 WORKDIR /project/build
 CMD cmake -S /project/source -B . -G Ninja -DCMAKE_BUILD_TYPE=Debug -DQGC_BUILD_TESTING=ON -DQGC_STABLE_BUILD=OFF ; \
 	cmake --build . --target all --config Debug ; \

--- a/deploy/docker/run-docker.sh
+++ b/deploy/docker/run-docker.sh
@@ -5,4 +5,4 @@
 set -e
 
 docker build --file ./deploy/docker/Dockerfile-build-ubuntu -t qgc-ubuntu-docker .
-docker run --cap-add SYS_ADMIN --device /dev/fuse --rm -v ${PWD}:/project/source -v ${PWD}/build:/project/build qgc-ubuntu-docker
+docker run --cap-add SYS_ADMIN --device /dev/fuse --security-opt apparmor:unconfined --rm -v ${PWD}:/project/source -v ${PWD}/build:/project/build qgc-ubuntu-docker

